### PR TITLE
Fix build/clippy errors and update CASR

### PR DESCRIPTION
**Pull request by addisoncrump**
There appears to be a few missing imports from CASR components. Upstream, this should be fixed by making sure the macro itself explicitly refers to the items it uses, but we can fix this downstream for now.

Otherwise, clippy is mad again.

### CODE REVIEW

**Comment by addisoncrump** [comment]
Needs #1376

File: libafl_qemu/src/lib.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@
     allow(clippy::useless_conversion)
 )]
 #![allow(clippy::needless_pass_by_value)]
+#![allow(clippy::needless_pass_by_ref_mut)]
```
Comment (domenukk) [inline libafl_qemu/src/lib.rs:10]: That's a false positive? Always?

**Comment by domenukk** [comment]
Somehow the fuzzers on ubuntu don't work?

File: libafl_qemu/src/lib.rs (hunk 1 above)
Comment (addisoncrump) [reply libafl_qemu/src/lib.rs:10]: In libafl_qemu, it is at multiple points. I'll make this statement-level allows.

**Comment by addisoncrump** [comment]
> Somehow the fuzzers on ubuntu don't work?

Yeah I have no idea on this one chief :joy:

**Pull request merged**